code/modules/paperwork/photography.dm

```diff
@@ -1,23 +1,288 @@
 /*	Photography!
  *	Contains:
+ *      Film
  *		Camera
- *		Camera Film
+ *      Polaroid
+ *		DSS Card
  *		Photos
  *		Photo Albums
  */
 
+/*********
+* camera *
+*********/
+/obj/item/weapon/storage/lockbox/camera
+	name = "camera"
+	icon = 'icons/obj/items.dmi'
+	desc = "A digital camera. 10 photos left."
+	icon_state = "camera"
+	item_state = "electropack"
+	w_class = 2.0
+	slot_flags = SLOT_BELT
+	var/list/matter = list("metal" = 2000)
+	var/on = 1
+	storage_slots = 1
+	can_hold = list("/obj/item/weapon/dss_card")
+	req_access = list(access_cent_commander)
+var/icon_on = "camera"
+var/icon_off = "camera_off"
+var/size = 3
+var/dss_inserted = 0
+var/photosleft = 0
+var/photosmax = 10
+var/locked = 1
+var/obj/item/card = /obj/item/weapon/dss_card

```

Comment:
These must be properly indented, otherwise you're defining a bunch of global variables.